testsuite/gnat2goto/tests/ranges/test.out

```diff
@@ -1 +1,3 @@
-VERIFICATION SUCCESSFUL
+[assertion.1] Range Check: FAILURE
+[assertion.2] Range Check: FAILURE
```

Comment:
It'd probably be better to have a test where both failure and successes happen, and that's also testing some things like casting into a larger or smaller type

Reply:
I've extended the existing test with a few more cases, making the range check pass as well. I'm not sure it cover all reasonable cases.